spec/spec.md

```diff
@@ -365,7 +534,7 @@ _Input Descriptors_ are objects used to describe the proofing inputs a Verifier
 
 #### Input Descriptor Objects
 
-_Input Descriptors_ are objects that describe what type of input data/credential, or sub-fields thereof, is required for submission to the Verifier. _Input Descriptor_ objects are composed as follows:
+_Input Descriptors_ are objects that describe what type of input data/credential, or sub-fields thereof, is required for submission to the Verifier. _Input Descriptor Objects_ are composed as follows:
 
   - The object ****MUST**** contain an `id` property, and if present, its value ****MUST**** be a unique identifying string that does not conflict with the `id` of another _Input Descriptor_ in the same _Presentation Definition_ object.
```

Comment:
```suggestion
  - The object ****MUST**** contain an `id` property. The value of the `id` property ****MUST**** be a unique identifying string that does not conflict with the `id` of another _Input Descriptor_ in the same _Presentation Definition_ object.
```

Reply:
I know this isn't a comment on a change you've recommended, but it needs fixing.